### ISSUE TITLE
Mask(logical 'and') Job only allows Onebit PNG as input for Source Image

### ISSUE THREAD

**Issue by malajvan** [DDMAL/Rodan#698]
Masking Job works when inputting a ONEBIT PNG as the source image but throws an error (from inside gamera) when inputting another type of png. The current job wrapper allows Source Image to be of various png types.
(This might be a problem with porting within the wrapper and not necessarily the gamera4 code itself. Need more testing and understanding of the job)
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/celery/app/trace.py", line 412, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/celery/app/trace.py", line 704, in __protected_call__
    return self.run(*args, **kwargs)
  File "/code/Rodan/rodan/jobs/base.py", line 773, in run
    retval = self.run_my_task(inputs, settings, arg_outputs)
  File "/code/Rodan/rodan/jobs/gamera_rodan/wrappers/masking.py", line 80, in run_my_task
    image_result = image_source.and_image(image_mask)
  File "/usr/local/lib/python3.7/dist-packages/gamera/plugins/logical.py", line 42, in __call__
    return _logical.and_image(self, other, in_place)
TypeError: The 'self' argument of 'and_image' can not have pixel type 'RGB'. Acceptable values are ONEBIT, ONEBIT, ONEBIT, ONEBIT, and ONEBIT.
```

**Comment by malajvan** [comment]
According to Gamera 4 [documentation](https://gamera.informatik.hsnr.de/docs/gamera-docs/combine.html#id7), `and_image` and `xor_image` only takes type ONEBIT instead of RGB and Greyscale as suggested in our job wrapper. I assume this is a misuse of Gamera logical functions, and since there were no user-reported issues I'll correct the job wrapper accordingly.

**Comment by malajvan** [comment]
Fixed by [PR#724](https://github.com/DDMAL/Rodan/pull/724)
